### PR TITLE
NOJIRA: Add badges to discover community soffits and portlets

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] documentation is changed or added

##### Description of change

https://groups.google.com/a/apereo.org/d/topic/uportal-dev/ZZlBH8wsVAM/discussion

<a href="https://github.com/search?q=topic%3Auportal+topic%3Asoffit&type=Repositories">
        Google Code Style
      </a>
      <a href="https://github.com/search?q=topic%3Auportal+topic%3Aportlet&type=Repositories">
        Google Code Style
      </a>

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,12 @@
       <a href="https://google.github.io/styleguide/javaguide.html">
         <img src="https://img.shields.io/badge/code_style-Google-green.svg?style=flat" alt="Google Code Style">
       </a>
+      <a href="https://github.com/search?q=topic%3Auportal+topic%3Asoffit&type=Repositories">
+        <img src="https://img.shields.io/badge/discover-soffits-blue.svg?style=flat" alt="Google Code Style">
```
Comment (apetro) [inline README.md]: Alt text should be something like "Discover Soffits".

File: README.md [hunk 2]
```diff
@@ -12,6 +12,12 @@
...
+      </a>
+      <a href="https://github.com/search?q=topic%3Auportal+topic%3Aportlet&type=Repositories">
+        <img src="https://img.shields.io/badge/discover-portlets-blue.svg?style=flat" alt="Google Code Style">
```
Comment (apetro) [inline README.md]: Alt text should be something like "Discover Portlets".

File: docs/README.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ below.
       <a href="https://google.github.io/styleguide/javaguide.html">
         <img src="https://img.shields.io/badge/code_style-Google-green.svg?style=flat" alt="Google Code Style">
       </a>
+      <a href="https://github.com/search?q=topic%3Auportal+topic%3Asoffit&type=Repositories">
+        <img src="https://img.shields.io/badge/discover-soffits-blue.svg?style=flat" alt="Google Code Style">
+      </a>
+      <a href="https://github.com/search?q=topic%3Auportal+topic%3Aportlet&type=Repositories">
+        <img src="https://img.shields.io/badge/discover-portlets-blue.svg?style=flat" alt="Google Code Style">
```
Comment (apetro) [inline docs/README.md]: Alt text should be something like "Discover Portlets".

File: docs/README.md [hunk 4]
```diff
@@ -18,6 +18,12 @@ below.
       <a href="https://google.github.io/styleguide/javaguide.html">
         <img src="https://img.shields.io/badge/code_style-Google-green.svg?style=flat" alt="Google Code Style">
       </a>
+      <a href="https://github.com/search?q=topic%3Auportal+topic%3Asoffit&type=Repositories">
+        <img src="https://img.shields.io/badge/discover-soffits-blue.svg?style=flat" alt="Google Code Style">
```
Comment (apetro) [inline docs/README.md]: Alt text should be something like "Discover Soffits".

File: README.md (hunk 1 above)
Comment (ChristianMurphy) [inline README.md]: Good catch, updated

File: README.md (hunk 2 above)
Comment (ChristianMurphy) [inline README.md]: Good catch, updated

File: docs/README.md (hunk 4 above)
Comment (ChristianMurphy) [inline docs/README.md]: Good catch, updated

File: docs/README.md (hunk 3 above)
Comment (ChristianMurphy) [inline docs/README.md]: Good catch, updated

**Pull request merged**